src/aiida/transports/util.py

```diff
@@ -86,3 +86,24 @@ def copy_from_remote_to_remote(transportsource, transportdestination, remotesour
     .. note:: it uses the method transportsource.copy_from_remote_to_remote
     """
     transportsource.copy_from_remote_to_remote(transportdestination, remotesource, remotedestination, **kwargs)
+
+
+async def copy_from_remote_to_remote_async(
```

Comment:
Is this required in the utils? I don't find any usage

Reply:
Not sure how it's used, tbh, probably by external plugins? so far I just provide the similar functionality as in `copy_from_remote_to_remote`

Reply:
okay something that might be cleaned up in the future but for this PR it does not make so much sense